packages/react-native/UPGRADING.md

```diff
@@ -0,0 +1,279 @@
+
+`@bugsnag/react-native` is a complete rewrite and has no backwards compatibility with `bugsnag-react-native`. Therefore, the upgrade steps essentially consist of these three steps:
+
+1. removing `bugsnag-react-native` and (its dependencies) from your project
+2. adding `@bugsnag/react-native` to your project
+3. updating any usage of Bugsnag throughout your codebase
+
+#### A quick note on supported versions of React Native
+
+The oldest version of React Native supported by `@bugsnag/react-native@7.3` is `0.60`. If your project is running an older version than this, you should upgrade that first, or continue to use `bugsnag-react-native` until such time as you can upgrade.
+
+### Removing `bugsnag-react-native`
+
+Since `@bugsnag/react-native` is an entirely new module, `bugsnag-react-native` should be removed first. It's not possible to run them both alongside one-another. So the replacement can't be done incrementally.
+
+Firstly, remove the npm package with `npm` or `yarn`:
+
+```sh
+npm rm bugsnag-react-native
+# or
+yarn remove bugsnag-react-native
+```
+
+This will remove Bugsnag from your `package.json`, `package-lock.json`/`yarn.lock` and from `node_modules`. On the iOS side of your project you need to re-run `pod install` for it to remove anything Bugsnag-related from the pods in your workspace.
+
+```sh
+cd ios
+pod install
+```
+
+At this point it would be sensible to clean your projects so that no intermediate build assets are left around.
+
+For iOS, open up the project in XCode `open ios/<ProjectName>.xcworkspace`, and from the menubar select `Product > Clean build folder`.
+
+For Android, run `cd android && ./gradlew clean`.
+
+### Adding `@bugsnag/react-native`
+
+Full instructions are available at the [React Native integration guide](https://docs.bugsnag.com/platforms/react-native/react-native#installation) but here is a summary.
+
+Install the new package from npm using your preferred command:
+
+```sh
+npm install --save @bugsnag/react-native
+# or
+yarn add @bugsnag/react-native
+```
+
+#### Android
+
+Add the following to your `android/app/build.gradle` to integrate Bugsnag into your Anroid project:
+
+```groovy
+apply from: "../../node_modules/@bugsnag/react-native/bugsnag-react-native.gradle"
+```
+
+#### iOS
+
+Run cocoapods again so that it picks up the new local dependency:
+
+```
+cd ios
+pod install
+```
+
+### Updating usage
+
+Full integration instructions are available on the [React Native integration guide](https://docs.bugsnag.com/platforms/react-native/react-native#installation). This section will focus on specific areas that have changed.
+
+Since this update involved a major bump to Bugsnag's JS, Android and iOS notifiers – bringing all of their APIs in sync, virtually every usage of Bugsnag's API in your application will need updating.
```

Comment:
Is there a more positive way to word this?

Reply:
Have attempted to be more cheery about this. Let me know what you think.